api/controller/contestants.go

```diff
@@ -16,7 +16,7 @@ func GetContestans(w http.ResponseWriter, r *http.Request) {
 
 	resp := render.New()
 
-	contestants, err := fetchContestansFromCSV(resp, w)
+	contestants, err := fetchContestansList(resp, w)
 	if err != nil {
 		fmt.Println(err)
```

Comment:
You would probably like to return an error to the user if you weren't able to get the data